.github/workflows/move-to-in-review.yml

```diff
@@ -0,0 +1,26 @@
+name: move-card-to-in-review
+
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, synchronize, edited]
+
+jobs:
+  extract-issue-numbers:
+    runs-on: ubuntu-latest
+    outputs:
+      issues: ${{ steps.extract.outputs.issues }}
+    steps:
+      - id: extract
+        uses: MatrixFrog/pr-extract-issues@2684b0f1d7b1671b7fb9ab8bfbb9b2b88983e6c5
+        with:
+          way: 'body'
+  
+  move-issues:
+    needs: extract-issue-numbers
+    runs-on: ubuntu-latest
+    steps:
+      - env: 
+          ISSUES: ${{ needs.extract-issue-numbers.outputs.issues }}
+        run: "echo $ISSUES"
```

Comment:
here, we want to invoke the github API (or use another marketplace Action that invokes the github API) to grab the relevant issues and move them to the In Review column on the Project